### PR TITLE
[docs] Add SSL config options for postgres

### PR DESCRIPTION
**Pull request by nytai**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [x] Documentation

### SUMMARY

Add some SSL documentation to PG docs

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### REVIEWERS
@craig-rueda

### CODE REVIEW

**Review by craig-rueda — APPROVED** [review]

**Comment by nytai** [comment]
😠 `fossa`

File: docs/installation.rst [hunk 1]
```diff
@@ -424,8 +562,36 @@ The connection string for PostgreSQL looks like this ::
 
     postgresql+psycopg2://{username}:{password}@{host}:{port}/{database}
 
-See `psycopg2 SQLAlchemy <https://docs.sqlalchemy.org/en/13/dialects/postgresql.html#module-sqlalchemy.dialects.postgresql.psycopg2>`_.
+Additional  may be configured via the ``extra`` field under ``engine_params``. 
+If you would like to enable mutual SSL here is a sample configuration: ::
+
```
Comment (dpgaspar) [inline docs/installation.rst:567]: `.. code-block:: json`

File: docs/installation.rst [hunk 2]
```diff
@@ -424,8 +562,36 @@ The connection string for PostgreSQL looks like this ::
...
+                    "sslmode": "require",
+                    "sslrootcert": "/path/to/root_cert"
+            }
+         }
+    }
+
+If the key ``sslrootcert`` is present the server's certificate will be verified to be signed by the same Certificate Authority (CA). 
 
+If you would like to enable mutual SSL here is a sample configuration: ::
+
```
Comment (dpgaspar) [inline docs/installation.rst:581]: .. code-block:: json

**Review by dpgaspar — APPROVED** [review]
Looks good, just a couple of nit's

**Review by dpgaspar — APPROVED** [review]

**Pull request merged**